manifest_staging/charts/csi-secrets-store-provider-azure/templates/provider-azure-installer-windows.yaml

```diff
@@ -26,6 +26,9 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           args:
             - --endpoint=unix://C:\\provider\\azure.sock
+            {{- if .Values.logFormatJSON }}
```

Comment:
Do we also want to add:
```
            {{- if .Values.logVerbosity }}
            - -v={{ .Values.logVerbosity }}
            {{- end }}
```

Reply:
thank you! Added it to the helm charts and finished rebase.